changefeedtrigger/changefeedSample.py

```diff
@@ -0,0 +1,91 @@
+from .settings import *
+from azure.storage.blob.changefeed import ChangeFeedClient
+from azure.storage.blob import BlobServiceClient
+import azure.functions as func
+import logging
+
+
+def main(mytimer: func.TimerRequest) -> None:
+
+    # create blob service client to access connection string
+    blob_service_client = BlobServiceClient.from_connection_string(conn_str=CONNECTION_STRING)
+    change_feed_client = ChangeFeedClient.from_connection_string(conn_str=CONNECTION_STRING)
```

Comment:
>CONNECTION_STRING [](start = 74, length = 17)

can we extract this to settings file?

Reply:
nvmd, I was reading the read me and it asks the user to go in to the file and edit con_str

---
In reply to: [469606983](https://github.com/wastore/azure-storage-samples-for-python/pull/7#discussion_r469606983) [](ancestors = 469606983)